arbnode/dataposter/data_poster.go

```diff
@@ -1235,8 +1202,8 @@ type QueueStorage interface {
 
 type DataPosterConfig struct {
 	RedisSigner            signature.SimpleHmacConfig `koanf:"redis-signer"`
-	ReplacementTimes       string                     `koanf:"replacement-times"`
-	BlobTxReplacementTimes string                     `koanf:"blob-tx-replacement-times"`
+	ReplacementTimes       []time.Duration            `koanf:"replacement-times"`
+	BlobTxReplacementTimes []time.Duration            `koanf:"blob-tx-replacement-times"`
```

Comment:
Add code to parse duration slice from environment variables like this PR did for string slices
https://github.com/OffchainLabs/nitro/pull/2407/files

Reply:
Thanks for catching this, also there was another bug in EndCommonParse, fixed that as well, please take a look.